src/containers/Board.tsx

```diff
@@ -0,0 +1,79 @@
+import React, {FC, useEffect, useState} from 'react';
+import {Button} from '@material-ui/core';
+import {Square} from '../components/Square';
+import {fillArray, getWinner, playAudio} from '../utils';
+import {GameStatus} from './GameStatus';
+
+interface IProps {
+  isMute: boolean;
+}
+
+const STORAGE = 'react-game/squares';
+
+export const Board: FC<IProps> = ({isMute}) => {
+  const [squares, setSquares] = useState<Array<string | null>>(fillArray);
+  const [xIsNext, setXIsNext] = useState<boolean>(true);
+  const [winner, setWinner] = useState<string | null>(null);
+  const [isTie, setIsTie] = useState<boolean>(false);
+
+  useEffect(() => {
+    const getStorage = localStorage.getItem(STORAGE);
+    if (getStorage) setSquares(JSON.parse(getStorage || ''));
+  }, []);
+
+  useEffect(() => {
+    const winner = getWinner(squares)?.winner || null;
+    if (winner) {
+      setWinner(winner);
+    } else {
+      setWinner(null);
+    }
+
+    const isGameTie = !!(squares.filter((el) => el).length === 9);
```

Comment:
Why do you need to transform it to a boolean if `===` operator already gives a boolean?